easybuild/easyconfigs/a/AFNI/AFNI-23.0.03-foss-2022a.eb

```diff
@@ -0,0 +1,50 @@
+# Updated to latest version and CMake
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'AFNI'
+version = '23.0.03'
+
+homepage = 'http://afni.nimh.nih.gov/'
+description = """AFNI is a set of C programs for processing, analyzing, and 
+displaying functional MRI (FMRI) data - a technique for mapping human brain activity."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'openmp': True, 'pic': True}
+
+source_urls = ['https://github.com/afni/afni/archive/']
+sources = ['AFNI_%(version)s.tar.gz']
+checksums = ['d278b8b0eaa707a33120d11018b7ba302484ccf8d34f1b2cf6ea11ff0dec37ed']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
```

Comment:
@sassy-crick wondering whether the `buildopts` from the previous versions are still needed?

Reply:
@verdurin from what I could see it is not required any more:
```
[  0%] Building C object src/CMakeFiles/pcor_objs.dir/configured_pcor/afni_pcor_byte.c.o
cd /dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/src && /apps/sw-eb/software/GCCcore/11.3.0/bin/gcc -DDYNAMIC_suffix=\".so\" -DREPLACE_XT -DDTYPE=byte -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/niml -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-build/nifti2 -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-src/nifti2 -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-build/znzlib -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-src/znzlib -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-build/nifticdf -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-src/nifticdf -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-build/niftilib -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/_deps/nifti_clib-src/niftilib -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/src/configured_pcor -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/3DEdge/src -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/f2c -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/src -I/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/rickr -O2 -ftree-vectorize -march=native -fno-math-errno -fopenmp -fPIC -DUSE_TRACING -DHAVE_ZLIB -DREAD_WRITE_64 -DLINUX2 -O3 -DNDEBUG -fPIC -MD -MT src/CMakeFiles/pcor_objs.dir/configured_pcor/afni_pcor_byte.c.o -MF CMakeFiles/pcor_objs.dir/configured_pcor/afni_pcor_byte.c.o.d -o CMakeFiles/pcor_objs.dir/configured_pcor/afni_pcor_byte.c.o -c /dev/shm/jsassman/AFNI/23.0.03/foss-2022a/easybuild_obj/src/configured_pcor/afni_pcor_byte.c
```
That said, please feel free to test it to make sure nothing is slipping through. I don't know the program so I might get it wrong.

One thing which I cannot work out is I find two files in my local `PYTHONPATH` which should not be there:
`afnipy.egg-link  easy-install.pth`
The content of them refers to the build directory. Do you or @zao have any idea if they are actually needed?
```
$ cat python3.10/site-packages/afnipy.egg-link
/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/python_scripts
$ cat python3.10/site-packages/easy-install.pth
/dev/shm/jsassman/AFNI/23.0.03/foss-2022a/afni-AFNI_23.0.03/src/python_scripts
```

Reply:
@zao would know much better than I would.